### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by unknownpioneer**
translated

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
A suggestion, as document should be translated as "文档".

**Comment by unknownpioneer** [comment]
> A suggestion, as document should be translated as "文档".

But they are commonly used in a website, aren't they? "页面“ is acceptable too I suppose.

**Comment by yin1999** [comment]
> > A suggestion, as document should be translated as "文档".
> 
> But they are commonly used in a website, aren't they? "页面“ is acceptable too I suppose.

I don't agree with this. A single page can contain _multiple_ documents.

File: files/zh-cn/web/html/element/meta/name/index.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ CSS 颜色调整规范（CSS Color Adjustment specification）定义了以下元
   - `only light`
     - : 表示文档*仅*支持浅色模式，也就是浅色背景色和深色前景色。按照规范，`only dark` _是无效的_。如果在文档不支持深色模式的情况下强迫其以深色模式进行渲染，会导致内容不可读。所以，在未经配置的情况下，所有主要浏览器均默认使用浅色模式。
 
-  For example, to indicate that a document prefers dark mode but does render functionally in light mode as well:
+  举个例子，为了指明该页面默认使用深色模式同时也支持浅色模式，代码如下：
```
Comment (unknownpioneer) [inline files/zh-cn/web/html/element/meta/name/index.md:56]:
```suggestion
  举个例子，为了指明该文档默认使用深色模式同时也支持浅色模式，代码如下：
```

Comment (unknownpioneer) [inline files/zh-cn/web/html/element/meta/name/index.md:56]:
```suggestion
  举个例子，为了指明该文档默认使用深色模式同时也支持浅色模式，代码如下：
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**